Hyrax_DockerHub_Installation.adoc

```diff
@@ -0,0 +1,48 @@
+= Docker Installation (Recommended) =
+:Miguel Jimenez-Urias <mjimenez@opendap.org>:
+{docdate}
+:numbered:
+:toc:
+
+This is the simplest way to install and use the latest release of Hyrax. 
+
+== Prerequisites
+
+. `Docker` is running in the background.
+. You have a data folder. In this guide, we will assume it is `~/tmp/data/` .
+. Your data is stored in HDF5/NetCDF4 format, csv, or any other file format for which Hyrax has a data Handler (https://www.opendap.org/software/hyrax-data-server/[see all supported file formats by Hyrax])
+. OSX or Linux platform.
+
+== Run Hyrax and serve data
+
+. **Open a terminal and get the latest snapshot of hyrax**
+
+```
+docker pull opendap/hyrax:snapshot
+```
+
+TIP: Do not use `hyrax:latest`. We use Travis for CI/CD and `hyrax:snapshot` is fully tested, and packs with the correct versions needed to build Hyrax. Using `latest`, ironically may pull an older version of Hyrax.
+
+This will download the latest stable, complete build of Hyrax. 
+[start=2]
+. **Run Hyrax and make your data available on port `8080`**
+
+```
+docker run -d -h hyrax -p 8080:8080 \
+--platform linux/amd64 \
```

Comment:
As far as I know:
> --platform linux/amd6

Is only required on an OSX docker deployment.

Reply:
👍🏽